sanity-io/structured-content-2022
# National 2 Narrow font

# National 2 Narrow font

## Intent

Start using the "National 2 Narrow" bold font for the font variations specified in the Figma sketches

## Description

Previously these font variatoins were just requesting the default `sans-serif` font on the system, because the font had not yet been purchased. Now it has.

Replaces the unused "Larsseit" font which was still hanging around from the original version of the site (currently on the `main` branch).

## Testing this PR

1. Open the [About page](https://structured-content-2022-web-git-national-2-narrow-font.sanity.build/about)
2. Verify that the font of the main heading ("About the Structured Content Conference") looks similar to the Figma sketches, or inspect the element with the browser's dev tools and verify that the font used (e.g. "Fonts" tab in Firefox' Inspector) is National 2 Narrow Bold
